Detectors/MUON/MID/Workflow/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+set(MODULE_NAME "MIDWorkflow")
+set(MODULE_BUCKET_NAME mid_workflow_bucket)
+
+O2_SETUP(NAME ${MODULE_NAME})
+
+set(SRCS
+        src/ClusterizerSpec.cxx
+        src/ClusterLabelerSpec.cxx
+        src/DigitReaderSpec.cxx
+        src/RecoWorkflow.cxx
+        src/TrackerSpec.cxx
+        src/TrackLabelerSpec.cxx
+        )
+
+set(LIBRARY_NAME ${MODULE_NAME})
+set(BUCKET_NAME ${MODULE_BUCKET_NAME})
+
+O2_GENERATE_LIBRARY()
+
+O2_GENERATE_EXECUTABLE(
+  EXE_NAME mid-reco-workflow
```

Comment:
Look fine, but the executables should start with o2, please rename to ``o2-mid-reco-workflow``

Reply:
btw, this should have been caught by the ensure-executable-naming-convention test. I will have to check why that was not the case...

Reply:
Hi Ruben,
ok thanks for the review: I changed the name (also in the README) and force pushed.
Cheers,
Diego